docs/resources/Sticker.md

```diff
@@ -0,0 +1,137 @@
+
+###### Sticker Structure
+
+| Field       | Type                                            | Description                                                                                                                                                  |
+| ----------- | ----------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| id          | snowflake                                       | id of the sticker                                                                                                                                            |
+| pack_id?    | snowflake                                       | for standard stickers, id of the pack the sticker is from                                                                                                    |
+| name        | string                                          | name of the sticker                                                                                                                                          |
+| description | string                                          | description of the sticker                                                                                                                                   |
+| tags        | string                                          | for guild stickers, the name of a unicode emoji representing the sticker's expression. for standard stickers, a comma-separated list of related expressions. |
+| asset       | string                                          | **Deprecated** previously the sticker asset hash, now an empty string                                                                                        |
+| type        | integer                                         | [type of sticker](#DOCS_RESOURCES_STICKER/sticker-object-sticker-types)                                                                                      |
+| format_type | integer                                         | [type of sticker format](#DOCS_RESOURCES_STICKER/sticker-object-sticker-format-types)                                                                        |
+| available?  | boolean                                         | whether or not the guild sticker is available                                                                                                                |
+| guild_id?   | snowflake                                       | id of the guild that owns this sticker                                                                                                                       |
+| user?       | [user](#DOCS_RESOURCES_USER/user-object) object | the user that uploaded the guild sticker                                                                                                                     |
```

Comment:
Is there a specific endpoint or gateway event that contains this? Both `/guilds/:guild_id/stickers` and `/guilds/:guild_id/stickers/:sticker_id` don't return this as far as I'm aware

Reply:
Nevermind, turns out I'm stupid.

Reply:
`/guilds/:guild_id/stickers` and `/guilds/:guild_id/stickers/:sticker_id` both seem to return this field if the bot making the request has MANAGE_EMOJIS (MANAGE_EMOJIS_AND_STICKERS) permission in the guild